agoric/contract/src/kreadKit.js

```diff
@@ -1215,7 +1215,7 @@ export const prepareKreadKit = (
             // Find store record based on wanted character
             const sellRecord = market.itemEntries.get(offerArgs.entryId);
             sellRecord ||
-              assert.fail(X`${errors.itemNotFound(offerArgs.entryId)}`);
+              assert.fail(errors.itemNotFound(offerArgs.entryId));
```

Comment:
Right here we pass the entryId of the item to be bought to the error message. This means the entryId will be returned to the makeOffer result. Is this a security vulnerability? My thought process was that you can already see this id either from the react context or the p2p viewer, so it doenst make a difference. If it is we can hide just he entryId though using the X tag with the quote tag